aten/src/ATen/native/native_functions.yaml

```diff
@@ -4010,7 +4010,14 @@
   device_check: NoCheck
   device_guard: False
   dispatch:
-    CompositeImplicitAutograd: narrow_tensor_symint
+    CompositeExplicitAutograd: narrow_tensor_symint
+
+- func: narrow_backward(Tensor grad_output, SymInt[] input_sizes, int dim, Tensor start, SymInt length) -> Tensor
```

Comment:
Do you need this? Can't you use `narrow_scatter()` ?

Reply:
> narrow_scatter

I checked and there is no `narrow_scatter` function like `slice_scatter`. So defining a `narrow_backward` should be the go-to way to add batching rules with `CompositeImplicitAutograd` like [this comment said](https://github.com/pytorch/pytorch/pull/99757#issuecomment-1526109834)?